app/template/rest.py

```diff
@@ -80,12 +80,12 @@ def validate_parent_folder(template_json):
 def create_template(service_id):
     fetched_service = dao_fetch_service_by_id(service_id=service_id)
     # permissions needs to be placed here otherwise marshmallow will interfere with versioning
-    permissions = [p.permission for p in fetched_service.permissions]
+    [p.permission for p in fetched_service.permissions]
```

Comment:
This line seems to be an  accidental holdover

Reply:
Weirdly when I remove that line the tests fail. Probably related to the comment on the line above. Will see if I can investigate and work out why this is happening

Reply:
I have done some digging into why this was happening.

Here’s a fix, which should be merged first: https://github.com/alphagov/notifications-api/pull/4048

Then in this PR we can remove the line which is seemingly doing nothing.